### DISCUSSION TITLE
Can I add `2xs` size on theme setting?

### DISCUSSION THREAD

**Discussion by borachoidev** [mantinedev/mantine#4707] [Q&A]
Hi, I want to add `2xs` radius size onto mantine theme setting.
but I got the type error .
How do I override mantine size type ?

I'm using 5.10.2 version.

<img width="868" alt="스크린샷 2023-08-30 오전 11 34 57" src="https://github.com/mantinedev/mantine/assets/66766189/8187e978-a632-4e24-9a19-7735fa413723">

**Comment by rtivital** [comment]
You can use `as any` with the configuration object, or store the value in `theme.other`
